packages/react-client/src/trackManager.ts

```diff
@@ -48,8 +53,24 @@ export class TrackManager implements GenericTrackManager {
     setCurrentTrackMiddleware(() => middleware);
   }
 
-  public startStreaming = async (simulcastConfig?: SimulcastConfig, maxBandwidth?: TrackBandwidthLimit) => {
-    if (this.currentTrackId) throw Error("Track already added");
+  function getCurrentTrack(): Track | null {
+    return getRemoteOrLocalTrack(tsClient, currentTrackId);
+  }
+
+  async function initialize(deviceId?: string) {
+    mediaManager?.start(deviceId ?? true);
+  }
+
+  async function stop() {
+    mediaManager?.stop();
+  }
+
+  function getType(): TrackMetadata["type"] {
+    return TRACK_TYPE_TO_DEVICE[mediaManager.getDeviceType()];
+  }
```

Comment:
**nitpick:** since this is a react hook, we could just do
```suggestion
  const deviceType = TRACK_TYPE_TO_DEVICE[mediaManager.getDeviceType()]
```